build-zfs-module/Containerfile

```diff
@@ -0,0 +1,30 @@
+FROM quay.io/fedora/fedora-coreos:stable as kernel-query
+#We can't use the `uname -r` as it will pick up the host kernel version
+RUN rpm -qa kernel --queryformat '%{VERSION}-%{RELEASE}.%{ARCH}' > /kernel-version.txt
+
+# Using https://openzfs.github.io/openzfs-docs/Developer%20Resources/Custom%20Packages.html
+FROM registry.fedoraproject.org/fedora:36 as builder
```

Comment:
A tricky thing here is this value is actually dependent on the version of Fedora in fedora-coreos...which is going to change Soon.

But...we can probably just fix that later.